server/views/pages/scan/scan-barcode-result.njk

```diff
@@ -18,16 +18,57 @@
       }) }}
     {% endif %}
 
+    {% if form.errorCode == null %}
+      {% include "partials/scan/scan-barcode-result-ok.njk" %}
+    {% endif %}
+    {% if form.errorCode.code == 'DUPLICATE' %}
+      {% include "partials/scan/scan-barcode-result-duplicate.njk" %}
+    {% endif %}
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds" id="scan-barcode-form">
+        <h2 class="govuk-heading-l">Scan another barcode</h2>
+        <img src="/assets/images/barcode-scanner.png" title="A barcode scanner being used to scan a barcode from an item of mail" />
+
+        <form action="scan-barcode" method="post" novalidate>
+          <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
+
+          {% call govukFieldset() %}
+
+            {{ govukInput({
+              classes: 'govuk-!-width-one-half',
+              label: {
+                text: 'Point the scanner at the barcode',
+                classes: 'govuk-visually-hidden'
+              },
+              attributes: {
+                autocomplete: 'off',
+                'data-inputmask': '"mask": "9999-9999-9999", "placeholder": " ", "autoUnmask": true, "removeMaskOnSubmit": true',
```

Comment:
Is this the scanner input rather than the manual input? In which case should we turn off the mask. I think they also wanted it to be hidden text like a password field 🤷